util/cron/test-perf.hpe-apollo-hdr.gasnet-ibv.playground.bash

```diff
@@ -32,10 +32,11 @@ if [[ "$SKIP_ML_PLAYGROUND" == "1" ]]; then
   exit
 fi
```

Comment:
Make sure to set `SKIP_ML_PLAYGROUND` to 0 (see line 29)

Reply:
Will do. Thanks!